editor/editor_properties.cpp

```diff
@@ -103,6 +107,7 @@ void EditorPropertyText::update_property() {
 	if (text->get_text() != s) {
 		int caret = text->get_caret_column();
 		text->set_text(s);
+		text->set_tooltip_text(s.left(TOOLTIP_MAX_LENGTH).strip_edges() + String((s.length() > TOOLTIP_MAX_LENGTH) ? "..." : ""));
```

Comment:
```suggestion
		text->set_tooltip_text(get_tooltip_string(s));
```
Unless something is special about this case